console/test.ts

```diff
@@ -1,6 +1,8 @@
 import fs = require('fs');
 import Unmarshaller = require('../src/unmarshal');
 import Interpreter = require('../src/interpreter');
+var async = require('async');
```

Comment:
You should use `import`, otherwise TypeScript will type `async` as the `any` type, and won't type check your code.

Since `async` is a JavaScript library, you'll need to add a `reference` comment at the top of the file that references the DefinitelyTyped typings for the `async` library:

```
/// <reference path="../bower/DefinitelyTyped/async/async.d.ts" />
// ^ at the top of the file

....

import async = require('async');
```